lib/knock/objects.rb

```diff
@@ -29,6 +29,23 @@ def get(collection:, id:)
         execute_request(request: request)
       end
 
+      # Retrieves paginated objects in a collection for the provided environment
+      #
+      # @param [Hash] options Options to pass to the paginated users endpoint query.
+      # These include:
+      # - page_size: size of page to be returned (max: 50)
+      # - after:  after cursor for pagination
+      # - before: before cursor for pagination
```

Comment:
Question: why aren't we enumerating out the options values for the subscription actions (they are the same set)?